Sami/Resources/themes/default/layout/layout.twig

```diff
@@ -55,11 +55,14 @@
             <form action="#" method="GET">
                 <select id="version-switcher" name="version">
                     {% for version in project.versions %}
-                        <option value="{{ path('../' ~ version ~ '/index.html') }}"{{ version == project.version ? ' selected' : '' }}>{{ version.longname }}</option>
+                        <option value="{{ path('../' ~ version ~ '/index.html') }}" data-version="{{version}}">{{ version.longname }}</option>
                     {% endfor %}
                 </select>
             </form>
         {% endif %}
+        <script>
+            $('option[data-version="'+window.projectVersion+'"]').attr('selected', true);
```

Comment:
Depending on jQuery version used you're better use `.prop` (sets DOM JS Object attribute) instead of `.attr` (sets HTML node attribute). Before `.attr` function did both things, but this was fixed and `.prop` function was introduced.